frontend/src/routes/__root.tsx

```
@@ -3,6 +3,8 @@ import { lazy } from 'react';
 
 import { defaultENV } from '@/envconfig';
 
+import Error from './@components/Error';
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)